connect_four/ai.rb

```diff
@@ -0,0 +1,38 @@
+require './board.rb'
+
+class AI
+  attr_reader :player_id
```

Comment:
Great use of reader, that way we don't accidentally goof and change the id partway thru.